tutorials/introductory/usage.py

```diff
@@ -714,7 +714,7 @@ def my_plotter(ax, data1, data2, param_dict):
 #
 #   plt.plot(x, y, markevery=10)
 #
-# The markevery argument allows for naive subsampling, or an
+# The ``markevery`` argument allows for naive subsampling, or an
```

Comment:
I think we do kwargs as `*markevery*`.

Reply:
I followed the formatting that was done on Line 708 and Line 712 for `markevery` in the paragraph. I can change all three instances to be formatted as *kwargs* as needed.